Tests/Unit/psgithub.tests.ps1

```diff
@@ -0,0 +1,69 @@
+#
+# This is a PowerShell Unit Test file.
+# You need a unit test framework such as Pester to run PowerShell Unit tests. 
+# You can download Pester from http://go.microsoft.com/fwlink/?LinkID=534084
+#
+
+$Global:ModuleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
+
+$OldLocation = Get-Location
+Set-Location -Path $ModuleRoot
+if (Get-Module PSGitHub -All)
```

Comment:
Please make sure that we use named parameters, and do not rely on parameter positions.